ducc/src/object.rs

```diff
@@ -154,6 +211,18 @@ impl<'ducc> Object<'ducc> {
     }
 }
 
+#[derive(Default)]
+pub struct PropertyDescriptor<'ducc, V>
+where
+    V: ToValue<'ducc> {
+    pub enumerable: bool,
+    pub configurable: bool,
+    pub writable: bool,
+    pub value: Option<V>,
```

Comment:
I made this `ToValue`. However this means that if there is no Value, you still need to define a unit type for the struct. Not sure if this is worth it.

Reply:
I was thinking the builder pattern could be used to avoid the extra type parameter but maintain the ergonomics. So you could do:

```rust
object.define_prop("key", PropertyDescriptor::new().enumerable(true).value("value here"));
```

Reply:
Ahh. I read that in your original comment and then somehow forgot about it immediately after. I'll look into that!